src/mlpack/methods/ann/layer/dropout.hpp

```diff
@@ -80,50 +77,48 @@ class Dropout
    * @param g The calculated gradient.
    */
   template<typename eT>
-  void Backward(const arma::Mat<eT>&& /* input */,
-                arma::Mat<eT>&& gy,
-                arma::Mat<eT>&& g);
+  void Backward(const arma::Mat<eT> && /* input */,
+                arma::Mat<eT> &&gy,
+                arma::Mat<eT> &&g);
 
   //! Get the input parameter.
-  InputDataType const& InputParameter() const { return inputParameter; }
+  InputDataType const &InputParameter() const { return inputParameter;  }
+
   //! Modify the input parameter.
-  InputDataType& InputParameter() { return inputParameter; }
+  InputDataType &InputParameter() { return inputParameter; }
 
   //! Get the output parameter.
-  OutputDataType const& OutputParameter() const { return outputParameter; }
+  OutputDataType const &OutputParameter() const { return outputParameter; }
+
   //! Modify the output parameter.
-  OutputDataType& OutputParameter() { return outputParameter; }
+  OutputDataType &OutputParameter() { return outputParameter; }
 
   //! Get the detla.
-  OutputDataType const& Delta() const { return delta; }
+  OutputDataType const &Delta() const { return delta; }
+
   //! Modify the delta.
-  OutputDataType& Delta() { return delta; }
+  OutputDataType &Delta() { return delta; }
 
   //! The value of the deterministic parameter.
   bool Deterministic() const { return deterministic; }
+
   //! Modify the value of the deterministic parameter.
-  bool& Deterministic() { return deterministic; }
+  bool &Deterministic() { return deterministic; }
 
   //! The probability of setting a value to zero.
   double Ratio() const { return ratio; }
 
   //! Modify the probability of setting a value to zero.
-  void Ratio(const double r)
-  {
+  void Ratio(const double r) {
```

Comment:
Avoid placing opening braces on the same line; see https://github.com/mlpack/mlpack/wiki/DesignGuidelines#brace-placement for more information.

Reply:
I am sorry for not being thorough with the style guidelines.  It is a bit different from what I am used to, but I will make sure it does not happen.

Reply:
No worries.